pkg/config/config.go

```diff
@@ -173,6 +173,12 @@ func GetOrLoadFromData(data []byte) (*V3ioConfig, error) {
 	return instance, failure
 }
 
+// update defaults when using config struct
+func GetOrLoadFromStruct(cfg *V3ioConfig) error {
```

Comment:
config instance is a singleton and should not be changed after initialisation.
`GetOrLoadFromStruct` suppose to return an existing or new configuration instance and optional error